kafka-connect-s3/src/main/java/io/confluent/connect/s3/TopicPartitionWriter.java

```diff
@@ -16,6 +16,7 @@
 
 package io.confluent.connect.s3;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Would you mind using `io.confluent.connect.storage.common.util.StringUtils;` instead. 

I know it's tricky because they have the same class name and ... they do the same thing :) But saves us one external dependency (even if commons.lang3 is already brought in).